README.md

```diff
@@ -272,4 +272,13 @@ long as a sufficient number of other witness nodes are timely producing blocks,
 is not necessary to use these flags once 128 blocks have been produced after the
 transition.
 
+# Tests
+
+To test `tinman`:
```

Comment:
The tests burped pretty hard on my machine. Something like this should be added to the readme. Maybe at the top, maybe in a notes section at the bottom, or maybe specifically in the test section

```
Tinman should work right out of the box, but on some, more delicately configured machines user's report `ijson` errors. Running `pip install ijson` or `pip3 install ijson` should take care of that.
```

Reply:
Added to #104